### DISCUSSION TITLE
Try to make two cy.click() inside the Cy.wrap(), but not able to do

### DISCUSSION THREAD

**Discussion by yuankez** [cypress-io/cypress#17612] [Questions and Help]
Current behavior:
There are 20 result box on the page, and when clicking on the box it will pop up a small modal expansion, and I have to click a cancel button to get back to main page. But when I use cy.wrap to iterate all result box, it dosent give me a chance to add another action after each click. And itt pop up an error saying "<div.results-page.bx--col-sm-4.bx--col-md-4>" is not visible after the first click() has been process. Can anyone help me take a look? And is that a cypress issue?

`        cy.get('@resultsPage').then(($resultsPage) =>{
            cy.wrap($resultsPage).click({multiple:true}).get('.bx--modal-footer.bx--btn-set').contains('Cancel').click();
        });`

**Comment by sandra-ouadghiri** [comment]
1. First get is useless. Wrap can call aliases. Just use a classical function, not a fat arrow.
2. You re not doing iterative but parallel clicking. Is that wanted? Not sure based on what you said. You should rather use a forEach() to do some action for each element.

resultsPage.forEach((plop) => {
  cy.wrap(plop).click()
  cy.get('.bx--modal-footer.bx--btn-set').contaisn('Cancel').click()
})

Searching for a class is not advised. Contains is not advised. Don't you have an proper attribute to query for?
